File: ICPC_10_4_16/penguins.cpp
#include <iostream>
#include <string>

int penguins[3]
ifstream myfile;
myfile.open ("input.txt");
getline(myfile,line)
while ( getline (myfile,line) )
{
	if(line[0] == "E")
		penguins[0]++;
	else if (line[0] == "M")
		penguins[1]++;
	else
		penguins[2]++;
}
myfile.close();


ofstream outputFile;
outputFile.open("output.txt");
outputString = "Emperor Penguin"
maxValue = 0;
for(int i =1;i<3;i++) {
	if(penguins[i]>maxValue){
		if(i == 1)
			outputString = "Macaroni Penguin"
		else
			outputString = "Little Penguin"
		maxValue = penguins[i];
	}
}

outputFile << outputString

File: ICPC_10_4_16/penguins.py
import sys
sys.stdin = open('input.txt', 'r')
sys.stdout = open('output.txt', 'w')

n = sys.stdin.readline()
penguins = {"Emperor":0, "Macaroni":0,"Little":0}
for i in range(int(n)):
	penguin = sys.stdin.readline().rstrip().split()
	if penguin[0][0] == 'E':
		penguins["Emperor"] = penguins["Emperor"] + 1
	elif(penguin[0][0] == "M"):
		penguins["Macaroni"] = penguins["Macaroni"] + 1
	elif(penguin[0][0] == "L"):
		penguins["Little"] = penguins["Little"] + 1

print max(penguins, key = lambda i:penguins[i]) + " Penguin"